pkg/models/node_approval.go

```diff
@@ -0,0 +1,39 @@
+package models
+
+import "fmt"
+
+// NodeApproval is used to denote the approval status of a given
+// node. These values are set based on the approval process for
+// nodes which will auto-approve some, rely on human approval for
+// others as well as rejecting some nodes as inelligible.
+type NodeApproval string
+
+const (
```

Comment:
why string instead of iota. I am asking because I keep asking myself the same question, specially that we always marshal to strings when persisting or sending fields

Reply:
One value might having a default value other than blank?